src/app/rundown-view/components/minishelf/mini-shelf.component.html

```diff
@@ -11,7 +15,7 @@
         (error)=handleMissingImage($event)
     />
     <div class="c-mini-shelf__note">
-        <span class="c-mini-shelf__title">{{ getSanitizedTitle() | uppercase }}</span>
+        <span class="c-mini-shelf__title">{{ getSanitizedTitle() | uppercase }} {{ showOnAirBorder ? 'ON-AIR' : ''}} {{ showNextBorder ? 'NEXT' : ''}}</span>
```

Comment:
We should not show with text that a minishelf is either `ON AIR` ot `NEXT`.
So this needs to be removed again.